ethcore/src/state/mod.rs

```diff
@@ -952,8 +951,58 @@ impl<B: Backend> State<B> {
 		}))
 	}
 
+	#[cfg(feature="evmbin-only")]
+	/// Populate a PodAccount map from this state.
+	/// Warning this is not for real time use
+	pub fn to_pod_slow(&self) -> PodState {
+		use ethereum_types::H160;
+
+		assert!(self.checkpoints.borrow().is_empty());
+		assert!(self.factories.trie.is_fat());
+
+		let mut result = BTreeMap::new();
+
+		let trie = match self.factories.trie.readonly(self.db.as_hashdb(), &self.root) {
+			Ok(trie) => trie,
+			_ => {
+				trace!(target: "state", "to_pod_slow: Couldn't open the DB");
+				return PodState::from(result);
+			}
+		};
+
+		let iter = match trie.iter() {
+			Ok(iter) => iter,
+			_ => return PodState::from(result),
+		};
+
+		let from_rlp = |b: &[u8]| Account::from_rlp(b).expect("decoding db value failed");
+
+		for item in iter {
+			if let Ok((addr, dbval)) = item {
+				let mut account = from_rlp(&dbval[..]);
+				if account.code_size().is_none() {
+					let addr_hash = account.address_hash(&H160::from(&addr[..]));
+					let accountdb = self.factories.accountdb.readonly(self.db.as_hashdb(), addr_hash);
+					let _ = account.cache_code(accountdb.as_hashdb());
+				}
+				result.insert(Address::from_slice(&addr), PodAccount::from_account(&account));
+			}
+		}
+
+		PodState::from(result)
+	}
+
+
+	/// Only for use with tests (to slow for real use),
+	/// panic if feature 'evmbin-only' is not activated.
+	#[cfg(not(feature="evmbin-only"))]
```

Comment:
Why do we need an explicit panic here? We should be able to just remove this section all together if `to_pod_slow` is indeed not used elsewhere.

Reply:
Only way, is to move call in evmbin, makes thing a bit more complicated. In fact it removes the need for a 'do_dump' fn in informant trait, it is better.

Reply:
Didn't get you on this part. Isn't evmbin always built with this feature on? And if this function is not used anywhere else, I think we can remove it without getting any compiling errors?

Reply:
The function was call from EvmTestClient (in ethcore module). EvmTestClient is used in other context than evmbin only (json test only at the time, plus libfuzzer, and anything because public). And the function call depend on a trait function (could not be in the evmbin informant trait cause this one is dropped at the time of the function call). So I needed an alternate implementation to compile. Anyway, with last commit it is not needed anymore.